### PR TITLE
added users documentation

### PR DESCRIPTION
**Pull request by robyncuppen**

### CODE REVIEW

File: blueprint/users/users.mson [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
++ feedCompleted: true (boolean)
++ website: `http://twitter.com/vikhyatk`
++ avatar (object)
+    + tiny: `https://media.kitsu.io/users/avatars/1/tiny.jpg?1434087646`
+    + small: `https://media.kitsu.io/users/avatars/1/small.jpg?1434087646`
+    + medium: `https://media.kitsu.io/users/avatars/1/medium.jpg?1434087646`
+    + large: `https://media.kitsu.io/users/avatars/1/large.jpg?1434087646`
+    + original: `https://media.kitsu.io/users/avatars/1/original.jpg?1434087646`
++ coverImage (object)
+    + tiny: `https://media.kitsu.io/users/cover_images/1/small.jpg`
```
Comment (wopian) [inline blueprint/users/users.mson:36]: ❌ Missing cache busting (`?1414038335`) for the cover images

**Review by wopian — CHANGES_REQUESTED** [review]

File: blueprint/users/users.mson (hunk 1 above)
Comment (robyncuppen) [reply blueprint/users/users.mson:36]: screen shot 2017-06-22 at 17 09 12

This is what I got from the api. I don't miss anything

Comment (wopian) [reply blueprint/users/users.mson:36]: Its gone again 😕

File: blueprint/users/users.mson [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+            + self: `https://kitsu.io/api/edge/users/1/relationships/favorites`
+            + related: `https://kitsu.io/api/edge/users/1/favorites`
+    + reviews (object)
+        + links (object)
+            + self: `https://kitsu.io/api/edge/users/1/relationships/reviews`
+            + related: `https://kitsu.io/api/edge/users/1/reviews`
+    +  stats (object)
+        + links (object)
+            + self: `https://kitsu.io/api/edge/users/1/relationships/stats`
+            + related: `https://kitsu.io/api/edge/users/1/stats`
```
Comment (wopian) [inline blueprint/users/users.mson:106]: New relationship added

```json
"oneSignalPlayers": {
  "links": {
    "self": "https://kitsu.io/api/edge/users/1/relationships/one-signal-players",
    "related": "https://kitsu.io/api/edge/users/1/one-signal-players"
  }
}
```

**Review by wopian — APPROVED** [review]

**Pull request merged**